shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -1226,6 +1226,9 @@ public void detachFromFlutterEngine() {
 
     if (flutterImageView != null) {
       flutterImageView.closeImageReader();
+      // Remove the FlutterImageView that was previously added by {@code convertToImageView} to
+      // avoid leaks when this FlutterView is reused later.
```

Comment:
When is it reused? :) an example here helps

Reply:
Done.